### PR TITLE
Suppress feature warning

### PR DESCRIPTION
**Pull request by okuta**
This PR improve test output readability.
Suppress feature warning that is printed by experimental function.

### CODE REVIEW

**Comment by delta2323** [comment]
It seems this PR is made on top of #2405. So should I check #2405 first?

**Comment by okuta** [comment]
Yes, please.

**Comment by delta2323** [comment]
As #2405 is merged, please merge the latest master branch if necessary.

**Comment by delta2323** [comment]
`chainer.links.TheanoFunction` is also experimental. Please annotate its test with `@testing.suppress_feature_warning()`

**Comment by okuta** [comment]
@delta2323 I fixed.

File: chainer/testing/helper.py [hunk 1]
```diff
@@ -29,3 +30,43 @@ def with_requires(*requirements):
...
+        self._warn.__exit__(None, None, None)
+        original_teardown(self)
+
+    klass.setUp = setUp
+    klass.tearDown = tearDown
+
+    return klass
+
+
+def suppress_feature_warning():
```
Comment (delta2323) [inline chainer/testing/helper.py]: Is feature should be future?

File: chainer/testing/helper.py [hunk 2]
```diff
@@ -29,3 +30,43 @@ def with_requires(*requirements):
...
+    """Decorator that suppress feature warning.
```
Comment (delta2323) [inline chainer/testing/helper.py]: ditto (feature -> future?)

**Comment by delta2323** [comment]
It seems to me chainer-test/#209 also tries to solve the same problem. I think it is better if we can solve it with environment variables because it is easier and we need not to define new functions.

Could you check the PR chainer-test/#209 to see if my understanding is correct and we can apply the same changes to Travis as well?

**Comment by okuta** [comment]
OK, I understand.
That is good idea.

**Pull request closed without merging**